docs/source/user_guide/box_model/index.rst

```diff
@@ -1,13 +1,61 @@
 Box model
 =========
+This section details the components of the box model, including reactions, mechanisms, conditions, and solutions. To use the MusicBox model, import::
+    
+    from acom_music_box import MusicBox, Conditions
 
-This section details the components of the box model, including reactions, mechanisms, conditions, and solutions.
+Using the previously defined in-code mechanism (see :ref:`Defining chemical species <species>`), a box model can now be created and initialized with it::
+    
+    box_model = MusicBox()
+    box_model.load_mechanism(mechanism)
 
-.. toctree::
-   :maxdepth: 1
+Initial and evolving conditions
+--------------------------------
+Use `Conditions` to set environmental parameters and species concentrations. Initial conditions define the environment that 
```

Comment:
We should be able to link directly to the Conditions object in the API reference. I think this may work
```suggestion
Use :class:`acom_music_box.conditions.Conditions` to set environmental parameters and species concentrations. Initial conditions define the environment that 
```

Reply:
and any other time we mention objects that are defined within music box (and maybe musica), we may want to do something similar